packages/web/src/analytics/analytics.ts

```diff
@@ -2,41 +2,30 @@ import Cookies from 'js-cookie'
 import getConfig from 'next/config'
 import { isBrowser } from 'src/utils/utils'
 
-let analytics: {
-  track: (key: string, properties?: object, options?: object) => void
-  page: () => void
-}
+const { publicRuntimeConfig } = getConfig()
 
-let segmentPromise
+let ReactGA
 
 export const ALLOW_ANALYTICS_COOKIE_NAME = '__allow__analytics__cookie__'
 export const RESPONDED_TO_CONSENT = '__responded_to_consent__'
 
+const OPTIN_EXPIRE_DAYS = 365
+const OPTOUT_EXPIRE_DAYS = 1
+
 export async function canTrack() {
-  return !!Cookies.get(ALLOW_ANALYTICS_COOKIE_NAME)
+  const allowTrack = await Cookies.get(ALLOW_ANALYTICS_COOKIE_NAME)
+  return isBrowser() && !!allowTrack && publicRuntimeConfig.ENV !== 'development'
 }
 
 export async function showVisitorCookieConsent() {
   return !Cookies.get(RESPONDED_TO_CONSENT)
 }
 
 export async function initializeAnalytics() {
-  if (!isBrowser() || !(await canTrack())) {
-    return {
-      track: noTrack,
-      page: noTrack,
-    }
-  }
+  ReactGA = await import('react-ga').then((mod) => mod.default)
 
-  if (!segmentPromise) {
-    segmentPromise = import('load-segment').then((mod) => mod.default)
-  }
-  if (!analytics) {
-    const Segment = await segmentPromise
-    const { publicRuntimeConfig } = getConfig()
-    analytics = Segment({ key: publicRuntimeConfig.__SEGMENT_KEY__ })
-  }
-  return analytics
+  ReactGA.initialize(publicRuntimeConfig.GA_KEY)
+  ReactGA.pageview('initial pageview')
```

Comment:
I think we can remove the initial page view as it was just for testing right?

Reply:
I was originally going to remove it, but then I realized we only track page views on route change.  Thought it would be useful to have analytics for the initial visit.
```
    this.props.router.events.on('routeChangeComplete', async () => {
      await analytics.page()
    })
```

Reply:
Although, I should add a `canTrack` check to that as well...

Reply:
Oh all evidence ive seen shows that page loads are being tracked even on initial load. 

and if that is not the case id like like to not have the call to track happen inside of the initialize function as it should be idempotent so that we can call it in track function to ensure analytics are available.

Reply:
I could be wrong, but I'm not seeing any analytics pop up on page load other than this "initial pageview".  Where would you like to have the call happen then?

Reply:
ok if we need to add, I think the best spot would be in side of `_app.tsx` in the componentDidMount function. right after it calls initialize.